backend/app/views/spree/admin/variants/_form.html.erb

```diff
@@ -97,6 +94,19 @@
                                   { class: 'custom-select fullwidth' } %>
         </div>
       </div>
+
+      <div class="col-3">
+        <div class="field" data-hook="shipping_category">
+          <%= f.label :shipping_category %>
+          <%= f.field_hint :shipping_category %>
+          <%= f.collection_select :shipping_category_id,
+                                  @shipping_categories,
+                                  :id,
+                                  :name,
+                                  { include_blank: t('.use_product_shipping_category') },
```

Comment:
Is this translation missing?

Reply:
Maybe. But this shouldn't be related to api key specs, no?

Reply:
No, unrelated but still something to address 🙂

Reply:
Well, it defaults to what we want, and the i18n test task is not complaining, so it's up to you. Are we not being explicit with the translations somewhere else?

Reply:
Done